src/Supermarket.API/Mapping/ResourceToModelProfile.cs

```diff
@@ -12,7 +12,7 @@ public ResourceToModelProfile()
             CreateMap<SaveCategoryResource, Category>();
 
             CreateMap<SaveProductResource, Product>()
```

Comment:
Hi. This change does not work when testing the application:

![image](https://user-images.githubusercontent.com/15124905/196568555-1c9afed5-cfc0-4f9c-adfd-fbad0ba5eb3d.png)

_.GetHashCode()_ produces an unpredictable number that varies depending on the machine you're running the application, and this number does not match any valid enum value. We still need an explicit cast to the respective enum type here.